Game.java

```diff
@@ -96,7 +77,7 @@ public void pickGameMode() {
                 return;
             }
         }
-        print("You did not pick a valid gamemode.");
+        consoleGame.invalidGamemode();
```

Comment:
From UX perspective, we wouldn't want the game to terminate because of an invalid menu choice.
How can you get this to notify they user they have entered an invalid value and must choose again?
As mentioned already, this is a Console concern so should be in ConsoleGame.